client/js/main.js

```diff
@@ -13,6 +13,25 @@ let urlProperties = {};
 let maxZ = {};
 export const dropTargets = new Map();
 
+function compareDropTarget(widget, t, exclude){
+  const dropTarget = asArray(t.get('dropTarget'))
+  let isValid = false;
+  for(let i = 0; i < dropTarget.length; i++) {
+    let isValidObject = true;
+    for(const key in dropTarget[i]) {
+      if(dropTarget[i][key] != widget.get(key) && (exclude == true || (key != 'type' || widget.get(key) != 'deck' || dropTarget[i][key] != 'card'))) {
+        isValidObject = false;
+        break;
+      }
+    }
+    if(isValidObject) {
+      isValid = true;
```

Comment:
I think you can just `return true` here and get rid of `isValid`. But this is also fine.